packages/react/src/patterns/blocks/CalloutWithMedia/__tests__/CalloutWithMedia.test.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import CalloutWithMedia from '../CalloutWithMedia';
+import React from 'react';
+import { shallow } from 'enzyme';
+
+describe('CalloutWithMedia', () => {
+  it('renders as expected', () => {
+    const contentblocksimple = {
+      copy: 'Lorem  ipsum dolor sit amet',
+      heading: 'Lorem  ipsum dolor sit amet',
+    };
+
+    const callout = shallow(
+      <CalloutWithMedia contentblocksimple={contentblocksimple} />
```

Comment:
Your test here might fail. I'm also wondering if this test is even necessary if we're doing storyshots already...

Reply:
It passed CircleCi, you think I can delete the test file?